### PR TITLE
DeveloperGuide.adoc : Added Instructions Manual Testing

### LINKED ISSUE #75: Update Instructions for Manual Testing part for Developer Guide

### PR DESCRIPTION
**Pull request by Amoscheong97**

### CODE REVIEW

**Review by Exeexe93 — CHANGES_REQUESTED** [review]
Please also edit line 410 :

   Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.

Change "a set of sample contacts" to "no orders".

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -429,12 +436,115 @@ _{ more test cases ... }_
 .. Other incorrect delete commands to try: `delete`, `delete x` (where x is larger than the list size) _{give more}_ +
    Expected: Similar to previous.
```
Comment (khsc96) [inline docs/DeveloperGuide.adoc:433]: Please update the deleting a person developer guide. Our delete is not by id is by transaction_id

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -429,12 +436,115 @@ _{ more test cases ... }_
...
+=== Mark order as Done
+
+. Mark order as done whenever an order transaction is completed. The correct syntax should be as follows: done <TRANSACTION ID>
+
+.. Test case: `done 83z7ehaq` +
+   Expected: The specific order with transaction ID 83z7ehaq will be moved from the current “Today’s Order” list to the list of completed orders.
+.. Test case: Invalid syntax
+   Expected: No order is marked as done. The error message will be displayed on the error response box describing the error.
+
+=== Import orders
```
Comment (khsc96) [inline docs/DeveloperGuide.adoc]: can swap with clear command to have a better flow

**Review by khsc96 — CHANGES_REQUESTED** [review]
Overall well done, please make the few changes and we are good to merge!

**Pull request merged**